webapp/api/launchpad.py

```diff
@@ -0,0 +1,6 @@
+from launchpadlib.launchpad import Launchpad
+
+
+launchpad = Launchpad.login_anonymously(
+    "snappy", "production", version="devel"
```

Comment:
is this an account: `snappy`?

Reply:
From the launchpadlib docs: "The first argument to Launchpad.login_anonymously() is a string that identifies the web service client".

Reply:
then use `storefront` instead